### PR TITLE
Launch beta.vyperlang.org

### LINKED ISSUE #1: Launch beta.vyperlang.org

### PR DESCRIPTION
**Pull request by DanielSchiavini**
Closes #1 

The following is done:
- [x] Make the image/network/container names configurable
- [x] Update nginx configuration
- [x] Push changes to the server
- [x] Create .env file with
    - `HUB_NAME=jupyterhub-beta`
    - `NOTEBOOK_NAME=titanoboa-beta-notebook`
    - `REF=vyper-0.4.0`
- [x] Update CI scripts

However, I'll need @charles-cooper credentials for:
- [ ] Create new OAuth client or add redirect URL to existing OAuth client
- [x] Create DNS record
- [x] Generate HTTPS certificates
- [ ] Create GitHub Action environments (with variable `TRY_VYPERLANG_HOST`)

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
+# build the docker images for the project
+
+# load the environment variables
+set -o allexport; source .env; set +o allexport;
```
Comment (charles-cooper) [inline build.sh:5]: hmm, i don't think we should be touching `.env` directly

File: .github/workflows/restart.yaml [hunk 2]
```diff
@@ -19,11 +19,14 @@ jobs:
...
       - name: Run command
         shell: bash
         run: |
             ssh try_vyper bash <<EOF
-            cd try.vyperlang.org
-            docker compose down
-            docker compose up --detach
+            cd ~/try.vyperlang.org
+            ./restart.sh
+            cd ~/beta.vyperlang.org
```
Comment (charles-cooper) [inline .github/workflows/restart.yaml:30]: shouldn't this be handled by environments?

Comment (DanielSchiavini) [reply .github/workflows/restart.yaml:30]: See [line 22](https://github.com/vyperlang/try.vyperlang.org/blob/287f4528ef42c57c0bdf1e9bdaa7f54edfa4b808/.github/workflows/restart.yaml#L22)

File: build.sh (hunk 1 above)
Comment (DanielSchiavini) [reply build.sh:5]: If we use environment variables, it gets quite tricky to also run the batch files directly on the server

**Review by charles-cooper — APPROVED** [review]

**Comment by charles-cooper** [comment]
note: it would be good to have github actions automatically generate the env file from github actions vars

**Pull request merged**